### PR TITLE
Fix: Fixed issue where custom font wasn't applied to the details pane

### LINKED ISSUE #12027: Bug: Custom font not used in item preview pane
### Description

image
The underlined bits of text are not using the font set in the Settings file

### Steps To Reproduce

1. Set a custom font
2. Open the preview pane

### Requirements

A custom font to be set

### Files Version

2.4.61.0

### Windows Version

10.0.25336.1000

### Log file

[debug.log](https://github.com/files-community/Files/files/11185352/debug.log)

### PR DESCRIPTION
**Pull request by QuaintMako**
**Resolved / Related Issues**
- [x] Were these changes approved in an issue or discussion with the project maintainers? In order to prevent extra work, feature requests and changes to the codebase must be approved before the pull request will be reviewed. This prevents extra work for the contributors and maintainers.
   Closes #12027 

**Validation**
How did you test these changes?
- [x] Did you build the app and test your changes?
- [ ] Did you check for accessibility? You can use Accessibility Insights for this.
- [ ] Did you remove any strings from the en-us resource file?
   - [ ] Did you search the solution to see if the string is still being used? 
- [ ] Did you implement any design changes to an existing feature?
   - [ ] Was this change approved?
- [ ] Are there any other steps that were used to validate these changes?
   1. Open app ...
   2. Click settings button ...

**Screenshots**

PreviewFonts

### CODE REVIEW

File: src/Files.App/UserControls/Pane/PreviewPane.xaml [hunk 1]
```diff
@@ -207,12 +211,12 @@
 					HorizontalAlignment="Center"
 					FontSize="20"
 					FontWeight="Bold"
```
Comment (Jay-o-Way) [inline src/Files.App/UserControls/Pane/PreviewPane.xaml:213]: Is there a StaticResource for this?

**Review by Jay-o-Way — COMMENTED** [review]
Always good to not-hardcode :⁠-⁠)

File: src/Files.App/Constants.cs [hunk 2]
```diff
@@ -103,6 +103,11 @@ public static class UI
 			public const double ContextMenuItemsMaxWidth = 250;
 		}
 
+		public static class Appearance
+		{
+			public const string StandardFont = "Segoe UI Variable";
```
Comment (yaira2) [inline src/Files.App/Constants.cs:108]: Is this present on Windows 10?

File: src/Files.App/Helpers/AppThemeResourcesHelper.cs [hunk 3]
```diff
@@ -25,22 +25,22 @@ public static void LoadAppResources(this IResourcesService service, IAppearanceS
...
 				appearance.AppThemeSidebarBackgroundColor = ""; //migrate to new default
 
-			if (!String.IsNullOrWhiteSpace(appThemeFileAreaBackgroundColor) && appThemeAddressBarBackgroundColor != "#00000000")
+			if (!string.IsNullOrWhiteSpace(appThemeFileAreaBackgroundColor) && appThemeAddressBarBackgroundColor != "#00000000")
 				service.SetAppThemeFileAreaBackgroundColor(ColorHelper.ToColor(appThemeFileAreaBackgroundColor).FromWindowsColor());
 			else
 				appearance.AppThemeFileAreaBackgroundColor = ""; //migrate to new default
 
-			if (appThemeFontFamily != "Segoe UI Variable")
+			if (appThemeFontFamily != Constants.Appearance.StandardFont)
```
Comment (yaira2) [inline src/Files.App/Helpers/AppThemeResourcesHelper.cs:43]: I typically only create a constant if it's going to be reused

File: src/Files.App/UserControls/Pane/PreviewPane.xaml (hunk 1 above)
Comment (yaira2) [reply src/Files.App/UserControls/Pane/PreviewPane.xaml:213]: For the font family or something else?

Comment (Jay-o-Way) [reply src/Files.App/UserControls/Pane/PreviewPane.xaml:213]: Size+Weight

File: src/Files.App/Helpers/AppThemeResourcesHelper.cs (hunk 3 above)
Comment (0x5bfa) [reply src/Files.App/Helpers/AppThemeResourcesHelper.cs:43]: Using constants to reduce hard coding is a good reason.

Comment (QuaintMako) [reply src/Files.App/Helpers/AppThemeResourcesHelper.cs:43]: The reasoning is indeed to reduce hard coding.

Comment (yaira2) [reply src/Files.App/Helpers/AppThemeResourcesHelper.cs:43]: Should it at least be local to this helper?

**Review by yaira2 — APPROVED** [review]
LGTM

**Pull request merged**